src/index.css

```diff
@@ -69,7 +69,7 @@ html, body {
 }
 .glass-panel h1,
 .glass-panel h2 {
-  color: var(--orange);
+  /* color: var(--orange); */
```

Comment:
@LauraSchlueter22 If the color isn't being used, can you delete it? There's no need to leave dead code.